packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -33,18 +33,18 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
   // Set the desired control buttons from the meetings composite. particiapantsButton is always false since there is the peopleButton.
   let meetingCallControlOptions: MeetingCallControlOptions | undefined;
   let callControlsOptions: CallControlOptions | boolean;
-  if (typeof props.meetingCallControlOptions === 'boolean') {
+  if (typeof props.callControls === 'boolean') {
```

Comment:
Not _necessarily_ in this PR: This will read better if we (1) keep the logic for the two options separate, (2) use immutable local variables.

So:
```ts

const meetingCallControlOptions = inferMeetingCallControlOptions(props.callControls);
const callControlOptions = inferCallControlOptions(props.callControls);

....

const inferMeetingCallControlOptions = (callControls?: boolean | MeetingCallControlOptions): MeetingCallControlOptions | undefined {
    if (typeof callControls !== 'boolean') {
        return callControls;
    }
    if (callControls === true) {
        // I'm actually confused by this return value. Please add a comment why.
        return undefined;
    }
    
    // callControls === false
    return {
        chatButton: false,
        peopleButton: false,
    }
}

... etc.
```

Reply:
I will play with this as well. I dont think the logic quite matches but I agree that this is the better solution to pull the logic out for readability.